### DISCUSSION TITLE
Żądanie zawiera błędne dane. Skontaktuj się z autorem aplikacji.

### DISCUSSION THREAD

**Discussion by hluchy** [community/community#145542] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

I am adding a new product using the API in PHP.  A message will appear: "Żądanie zawiera błędne dane. Skontaktuj się z autorem aplikacji.". Is there a problem with your data or account? CLIENT_ID:  447cc1e547b740b994710ff3646f4fce.

{
  "name": "Luxusní kožená crossbody kabelka Marta, tmavě červená",
  "category": {
    "id": "5363"
  },
  "images": [
    {
      "url": "https://www.demix.cz/foto-eshop-produkty/103069-2024116180507465385.jpg"
    }
  ],
  "parameters": [
    {
      "id": "11323",
      "valuesIds": [
        "11323_1"
      ]
    }
  ],
  "description": {
    "sections": [
      {
        "items": [
          {
            "type": "Universální dámská kožená kapsa, kterou skvěle využijete vždy a všude. To je Marta."
          }
        ]
      }
    ]
  },
  "language": "cs-CZ"
}

**Comment by ebndev** [comment]
Hi @hluchy, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
